PopupMenu/PopupMenu.module.less

```diff
@@ -21,6 +21,11 @@
 		&.horizontal .body {
 			white-space: nowrap;
 			text-align: center;
+			margin-bottom: @agate-popupmenu-body-margin-bottom;
+
+			label {
```

Comment:
`.label`?

Reply:
I could not overwrite LabeledIconButton's label style by selecting the .label class, that is why I used the tag name.

Reply:
I think we should not define this property here. It could cause unexpected results if a user has some other `label` tag. I'd rather define this in `PopupMenu.module.less` from sampler. 
Such as,
```
.labeledIconButton {
	.applySkins({
		.label {
			color: white;
		}
	});
}
```
and pass `css` to the `LabeldIconButton` from the story.